cupy/_environment.py

```diff
@@ -369,6 +375,23 @@ def _preload_library(lib):
                 _log(f'Library {lib} could not be preloaded: {e}')
 
 
+def _get_cutensor_from_wheel(
+        version: str, cuda: str) -> List[str]:
+    """
+    Returns the list of shared library path candidates.
+    """
+    import pkg_resources  # defer import # NOQA
```

Comment:
With #2759 and #7003, perhaps we should not rely on `pkg_resources`?

Reply:
btw if PY37 is dropped (#7293) we can use `importlib.metadata` here.

Reply:
I suppose for this case, we can just compare the strings? `"2.0.0" >= "1.7.0"` should work. It might sound fragile, but should be ok for SemVer?

Reply:
Sorry I overlooked your reply :bow:
`import pkg_resources` emits DeprecationWarning so it should be avoided anyway.

Just comparing string works... until 2.10.0 is shipped 😄
```
>>> '2.10.0' > '2.9.0'
False
```

Reply:
https://github.com/cupy/cupy/pull/8012